content/en/blog/2023/otel-in-focus-06.md

```diff
@@ -0,0 +1,158 @@
+bug fix regarding the OpenTelemetry Collector version not displaying properly in
+the status field.
+
+0.78.0 includes enhancements such as updating various packages, support for
+scaling on Pod custom metrics, and improved config validation. Bug fixes include
+addressing issues related to prometheus relabel configs and setting the default
+go auto-instrumentation version correctly. Components that were updated include
+the OpenTelemetry Collector, OpenTelemetry Contrib, Java auto-instrumentation,
+.NET auto-instrumentation, Node.JS, Python, Go, and ApacheHTTPD.
+
+##### [Java](/docs/instrumentation/java/)
+
+[Version 1.27](https://github.com/open-telemetry/opentelemetry-java/releases/tag/v1.27.0)
+has made the Log Bridge API and SDK stable. Important changes include merging
+log-related contents into the appropriate artifacts, marking the log SDK
+artifact as stable, and changing the default value of otel.logs.exporter in
+opentelemetry-sdk-extension-autoconfigure. The release also includes various bug
+fixes and improvements for the API, SDK, metrics, exporters, testing, SDK
+extensions, and semantic conventions.
+
+Remember - the Log Bridge API is not intended for end-users. It is used to
+bridge existing appenders (e.g., Log4j, SLf4J, Logback, JUL) into OpenTelemetry.
+
+[Java Instrumentation](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/tag/v1.27.0)
+has been released, requiring some migrations to be performed.
+
+- Migration notes: changes in Jersey 3.0 instrumentation suppression keys,
+  renaming and splitting of opentelemetry-runtime-metrics artifact, deprecation
+  of InetSocketAddressNetServerAttributesGetter and
+  InetSocketAddressNetClientAttributesGetter, and introduction of new HTTP and
+  network semantic conventions.
+- New javaagent instrumentation for Quarkus RESTEasy Reactive and Reactor Kafka.
+- Enhancements including improvements in Micrometer bridge, Ktor
+  instrumentations, AWS SDK support, OkHttp 3, Jetty 11, Spring Boot, AWS Lambda
+  tracing, and type matching.
+- Bug fixes related to logback MDC instrumentation, kafka metrics reporter,
+  jetty context leak, filtering scalar Mono/Flux instances, and others.
+
+#### [Javascript](/docs/instrumentation/js/)
+
+[Version 1.14](https://github.com/open-telemetry/opentelemetry-js/releases/tag/v1.14.0)
+includes an optional `forceFlush` method to the `SpanExporter` interface.
+
+#### [.NET](/docs/instrumentation/net/)
+
+[Version 1.5.1](https://github.com/open-telemetry/opentelemetry-dotnet/releases/tag/core-1.5.1)
+has been released, as well as 1.5.0. 1.5.1 is a fix for issues introduced in
+1.5.0, and 1.5.0 consisted of the following major changes:
+
+- The bug introduced in 1.5.0-rc.1 that caused the "Build" extension to return
+  null has been fixed.
+- The Exemplars feature has been marked as internal and will be added back in
+  the 1.6.\* prerelease versions.
```

Comment:
What's the `\*` for? Were you planning to add a footnote?

Reply:
Looks like some escaping gone wild. Will fix.